lib/core/versions/latest/DocumentComposer.ts

```diff
@@ -124,32 +124,32 @@ export default class DocumentComposer {
    * Validates the schema of the given full document.
    * @throws SidetreeError if given document patch fails validation.
    */
-  public static validateDocument (document: any) {
+  private static validateDocument (document: any) {
     if (document === undefined) {
       throw new SidetreeError(ErrorCode.DocumentComposerDocumentMissing);
     }
 
-    const allowedProperties = new Set(['publicKeys', 'serviceEndpoints']);
+    const allowedProperties = new Set(['public_keys', 'service_endpoints']);
     for (let property in document) {
       if (!allowedProperties.has(property)) {
         throw new SidetreeError(ErrorCode.DocumentComposerUnknownPropertyInDocument, `Unexpected property ${property} in document.`);
       }
     }
 
     // Verify 'publicKeys' property if it exists.
```

Comment:
```suggestion
    // Verify 'public_keys' property if it exists.
```
or remove if you feel it's too trivial!

Same for all occurrences.